health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateQuestion.js

```
@@ -323,24 +327,51 @@ const CreateQuestion = ({ onSelect, className, level = 1, initialQuestionData, p
     maxWidth: "100rem"
   }
 
+  let dis = typeOfCall === "view" ? true : false;
   return (
     <React.Fragment>
       {initialQuestionData
-        ?.filter((i) => i.level === level && (i.parentId ? (i.parentId === parentId) : true))
+        ?.filter((i) => i.level === level && (i.parentId ? (i.parentId === parentId) : true)  && (i.level <= 3) && (i.isActive === true))
         ?.map((field, index) => {
           return (
             <Card type={"primary"} variant={"form"} className={`question-card-container ${className}`}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)